explorer/explorerroutes.go

```diff
@@ -597,10 +602,15 @@ func (exp *explorerUI) DecodeTxPage(w http.ResponseWriter, r *http.Request) {
 
 // Charts handles the charts displays showing the various charts plotted.
 func (exp *explorerUI) Charts(w http.ResponseWriter, r *http.Request) {
+	if exp.liteMode {
+		exp.StatusPage(w, "Activate full-functionality mode",
```

Comment:
Instead of `Activate full-functionality mode`, I'd prefer if this read `full-functionality mode required for this page` since nothing is broken or misconfigured globally, and it's optional.

Reply:
Fixed